src/content/docs/en/guides/content-collections.mdx

```diff
@@ -1,643 +1,181 @@
 ---
-title: Content Collections
+title: Collections
 description: >-
-  Content collections help organize your Markdown and type-check your
-  frontmatter with schemas.
+  Learn about Astro's content collections.
 i18nReady: true
 ---
-import { FileTree } from '@astrojs/starlight/components';
+import { FileTree, CardGrid, LinkCard, Steps } from '@astrojs/starlight/components';
 import Since from '~/components/Since.astro'
 import RecipeLinks from "~/components/RecipeLinks.astro"
 import Badge from "~/components/Badge.astro"
 
 <p><Since v="2.0.0" /></p>
 
-**Content collections** are the best way to manage and author content in any Astro project. Collections help to organize your documents, validate your frontmatter, and provide automatic TypeScript type-safety for all of your content.
+**Content collections** are the best way to manage sets of content in any Astro project. Collections help to organize and query your documents, enable Intellisense and type checking in your editor, and provide automatic TypeScript type-safety for all of your content.
+
+Astro v5.0 introduced the Content Layer API for querying content collections. This performant, scalable API provides built-in content loaders for your local collections. For remote content, you can use third-party and community-built loaders or create your own custom loader and pull in your data from any source.
```

Comment:
The Content Layer is more about defining collections. There are some runtime changes, but these are much smaller.

Reply:
added the word defining in my edit. Still up for further editing as necessary!